spec/motion-spec/matchers/respond_to_spec.rb

```diff
@@ -0,0 +1,34 @@
+# -*- encoding : utf-8 -*-
+describe "Matcher::RespondTo" do
+  it 'respond_to passes when subject responds to method name' do
+    expect("string").to respond_to(:upcase)
+  end
+
+  it "respond_to fails when subject doesn't respond to method name" do
+    expect_failure("\"string\" expected to respond to #floor"){ expect("string").to respond_to(:floor) }
+  end
+
+  it "respond_to when given number of arguments fails when subject doesn't respond to method name" do
+    expect_failure("\"string\" expected to respond to #floor with 2 arguments"){ expect("string").to respond_to(:floor).with(2).arguments }
+  end
+
+  it "respond_to when given number of arguments passes when subject responds to method name with exactly the same number of arguments" do
+    class TestCase; def call_me(a,b,c);   end
+end
+    expect(TestCase.new).to respond_to(:call_me).with(3).arguments
+  end
+
+  it "respond_to when given number of arguments fails when subject responds to method name with different number of arguments" do
```

Comment:
Line is too long. [129/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.